src/applications/vaos/utils/data.js

```diff
@@ -114,10 +114,10 @@ export function transformFormToExpressCareRequest(state) {
       facilityCode: facilityId,
       parentSiteCode: siteId,
     },
-    reasonForVisit: data.reasonForVisit,
-    additionalInformation: data.additionalInformation,
-    phoneNumber: data.phoneNumber,
-    verifyPhoneNumber: data.phoneNumber,
+    reasonForVisit: data.reasonForRequest.reason,
```

Comment:
Could you make these same updates in the mock response for the request submit in our api/index.js file?

Reply:
Looks like the api is passed the transformed request to use as mock data so I don't think we need to make any updates there.